frontend/test/metabase/scenarios/permissions/reproductions/19609-do-not-show-archived-collections.cy.spec.js

```diff
@@ -0,0 +1,32 @@
+import { restore } from "__support__/e2e/cypress";
+
+const UNARCHIVED_PARENT_NAME = "Unarchived parent";
+const ARCHIVED_NAME = "Archived child";
+
+describe("issue 19609", () => {
```

Comment:
#19609 is the PR that fixes the issue #19603. Btw, I've already created the repro for it yesterday.
https://github.com/metabase/metabase/pull/19615

Reply:
Thanks for pointing out, I've updated the issue number here https://github.com/metabase/metabase/pull/19609
Oh, sorry, I did not notice the repro yesterday. Should I remove the repro I created from the PR?